src/_config.yml

```diff
@@ -24,8 +24,8 @@ description: >- # this means to ignore newlines until "baseurl:"
   Write an awesome description for your new site here. You can edit this
   line in _config.yml. It will appear in your document head meta (for
   Google search results) and in your feed.xml site description.
-baseurl: "/civictechatlanta.org" # the subpath of your site, e.g. /blog
-url: "http://civictechatlanta.github.io/" # the base hostname & protocol for your site, e.g. http://example.com
```

Comment:
Not sure the best way to do these URL lines for production and local, but these lines are needed for the final version to work.

Reply:
Updated.
Found a way to use multiple config files - one for dev and one for production.
https://github.com/CivicTechAtlanta/civictechatlanta.org/pull/26